django/test/utils.py

```diff
@@ -384,6 +384,7 @@ class override_settings(TestContextDecorator):
     """
     def __init__(self, **kwargs):
         self.options = kwargs
+        self.enable_exception = None
```

Comment:
Define at the class level.

Reply:
As per https://github.com/django/django/commit/4ccf7154c3afdf45825857cb4249eb30744c9f74#r29449616

Reply:
> any reason why enable_exception can't be defined at the override_settings class level instead?

Nope, I will move it there.

> That feels odd storing storing a static None in override_settings().__dict__['enable_exception'] in the first place.

Interesting, any good reads/resources about some rules of thumb for putting things as class variables vs. instance variables? It didn't feel odd when I put it there :)